### ISSUE TITLE
Failed validation: #/definitions/QuoteResponse

### ISSUE THREAD

**Issue by gb221** [gadicc/node-yahoo-finance2#117]
I'm getting the same set of errors for each of the following tickers: `BITF.V`, `HUT.TO`, `MSTR`, and `NCTY` when calling:

```
await yahooFinance.quote(ticker)
```

An example set of errors for `NCTY` is:
```
    errors: [
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/required',
      keyword: 'required',
      params: [Object],
      message: "must have required property 'circulatingSupply'",
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/required',
      keyword: 'required',
      params: [Object],
      message: "must have required property 'coinImageUrl'",
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/required',
      keyword: 'required',
      params: [Object],
      message: "must have required property 'fromCurrency'",
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/required',
      keyword: 'required',
      params: [Object],
      message: "must have required property 'lastMarket'",
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/required',
      keyword: 'required',
      params: [Object],
      message: "must have required property 'startDate'",
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/required',
      keyword: 'required',
      params: [Object],
      message: "must have required property 'toCurrency'",
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/required',
      keyword: 'required',
      params: [Object],
      message: "must have required property 'volume24Hr'",
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/required',
      keyword: 'required',
      params: [Object],
      message: "must have required property 'volumeAllCurrencies'",
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCryptoCurrency/additionalProperties',
      keyword: 'additionalProperties',
      params: [Object],
      message: 'must NOT have additional properties',
      data: [Object]
    },
    {
      instancePath: '/0/quoteType',
      schemaPath: '#/definitions/QuoteCryptoCurrency/properties/quoteType/const',
      keyword: 'const',
      params: [Object],
      message: 'must be equal to constant',
      data: 'EQUITY'
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteCurrency/additionalProperties',
      keyword: 'additionalProperties',
      params: [Object],
      message: 'must NOT have additional properties',
      data: [Object]
    },
    {
      instancePath: '/0/quoteType',
      schemaPath: '#/definitions/QuoteCurrency/properties/quoteType/const',
      keyword: 'const',
      params: [Object],
      message: 'must be equal to constant',
      data: 'EQUITY'
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteEtf/additionalProperties',
      keyword: 'additionalProperties',
      params: [Object],
      message: 'must NOT have additional properties',
      data: [Object]
    },
    {
      instancePath: '/0/quoteType',
      schemaPath: '#/definitions/QuoteEtf/properties/quoteType/const',
      keyword: 'const',
      params: [Object],
      message: 'must be equal to constant',
      data: 'EQUITY'
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteEquity/additionalProperties',
      keyword: 'additionalProperties',
      params: [Object],
      message: 'must NOT have additional properties',
      data: [Object]
    },
    {
      instancePath: '/0',
      schemaPath: '#/definitions/QuoteMutualfund/additionalProperties',
      keyword: 'additionalProperties',
      params: [Object],
      message: 'must NOT have additional properties',
      data: [Object]
    },
    {
      instancePath: '/0/quoteType',
      schemaPath: '#/definitions/QuoteMutualfund/properties/quoteType/const',
      keyword: 'const',
      params: [Object],
      message: 'must be equal to constant',
      data: 'EQUITY'
    },
    {
      instancePath: '/0',
      schemaPath: '#/anyOf',
      keyword: 'anyOf',
      params: {},
      message: 'must match a schema in anyOf',
      data: [Object]
    }
  ]
```

**Comment by PythonCreator27** [comment]
Error confirmed. Thanks for the detailed error report. The yahoo finance API *has* changed their response schema. This is something very new. This error should happen for almost every symbol, and it seems to do so...

I just opened a PR to fix this: #118. In the meantime, there isn't really much that anyone can do. You could pull down my branch and follow the instructions in CONTRIBUTING.md to use that branch temporarily.

@gadicc We definitely need to accommodate for this in our tests. Maybe regenerate the test files every week or so.

**Comment by gadicc** [comment]
Thanks @gb221 for reporting this and @PythonCreator27 for your super fast fix!

@PythonCreator27, agreed.  Maybe a GitHub Action or something.  I wouldn't regenerate the test files but rather just run the tests without using the cache, with:

```bash
$ FETCH_DEVEL="nocache" yarn test
```

and see if any start breaking.

**However**, currently (recently? or since we now have so many tests), this exceeds Yahoo's limits and your IP gets banned.  I guess this will be easier once I finish #76.
